lib/components/home/artist_rails/__tests__/artist_card-tests.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import 'react-native'
+import React from 'react'
+import renderer from 'react-test-renderer'
+
+import ArtistCard from '../artist_card'
+
+jest.mock('../../../spinner.js', () => 'ARSpinner')
+jest.mock('../../../opaque_image_view.js', () => 'AROpaqueImageView')
+jest.mock('../../../switch_view.js', () => 'ARSwitchView')
+
+it('renders correctly', () => {
+  const tree = renderer.create(<ArtistCard artist={artistProps().artist} />).toJSON()
+  expect(tree).toMatchSnapshot()
+})
+
+it('hides bio when metadata missing correctly', () => {
+  const props = artistProps(false)
+  const artistComponent = new ArtistCard(props)
+  const metadata = artistComponent.renderMetadata()
+
+  expect(metadata).toHaveLength(2)
+  expect(metadata[0].props.children).toEqual(props.artist.name.toUpperCase())
+  expect(metadata[1].props.children).toEqual(props.artist.formatted_artworks_count)
+})
+
+const artistProps = (hasBio?: boolean = true) => {
+  return {
+    artist: {
+      __id: "QXJ0aXN0Omp1YW4tZ3Jpcw==",
+      formatted_artworks_count: "14 works, 3 for sale",
+      formatted_nationality_and_birthday: hasBio ? "Spanish, 1887–1927" : "",
```

Comment:
`""` is a falsy value in Javascript so it bypasses the if statement  [See here](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)